source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,103 @@
+{% endpullquote %}
+
+After adding roles to the controls and content in your webpage. The next thing to look over are `aria` attributes. For instance we use `aria-live` for our `ProgressBar` slider. This is because the `ProgressBar` is always updating when a video is playing.
+
+{% pullquote %}
+For a more complete list of [ARIA attributes see the specification](https://www.w3.org/TR/wai-aria-1.1/).
+{% endpullquote %}
+
+Finally you need to add an accessible "name" and "status" to an element so that it can be referred to correctly. For instance to indicate that the `MuteToggle` is not just a "button" we have to set the `title` and the `innerHTML`/`innerText` to "Mute Toggle". In Video.js we refer to accessible "name"/"status" updates as "ControlText". This will allow the screen reader to refer to the `MuteToggle` as "Mute Toggle" rather than button. We also have to update the button whenever it is "pressed" to indicate that it will now "Mute" or "Unmute" depending on its state.
+
+Here are some examples of accessiblity straight from Video.js:
+* The `MuteToggle` `<button>`:
+  * Has `aria-live` set to polite, which means that the screen reader should wait until the user is idle to send valuenow updates to the user
+  * Has "ControlText" of "Mute" or "Unmute" which indicates the current status of the button to the user. "ControlText" is a Video.js-ism that indicates a change in the `innerHTML` and `title` attribute that is localized along the way.
+* The `VolumeBar` slider `<div>`:
+  * Has a `role` attribute with a value of `slider` like this: `role="slider"`
+  * Has a `tabIndex` attribute
+  * Has `EventHandlers` that listen for:
+    * The up and right arrow keys to increase the volume and the slider percentage
+    * The down and left arrow keys to decrease the volume and the slider percentage
+  * Has `aria-label` of "volume level" which is an accessible name that the screen reader schould refer to this with
+  * Has `aria-valuenow` and `aria-valuetext` properties that update to indicate the current volume level (so the screen reader can read it)
+  * Has `aria-live` set to polite, which means that the screen reader should wait until the user is idle to send valuenow updates to the user
+
+### The problem and the solution
+
+Now let's talk about how screen reader accessibility broke in Video.js. First `VolumeMenuButton` replaced `MuteToggle` and `VolumeControl` on the `ControlBar`. `VolumeMenuButton` was set to mimic `MuteToggle` when clicked. It was would also show the `VolumeControl` on mouseover or focus. This was a problem because a `VolumeControl` was a now a child of a button, and buttons should not contain other controls. So to the screen reader and to the DOM there are two `MuteToggle button` controls. When visually there is a `VolumeControl` and a `MuteToggle`. Below you can see a gif of this behavior in action:
```

Comment:
- Maybe in the first sentence specify that it `broke in Video.js 5`?
- `It was would also show` should be `It would also show`